src/pages/community.tsx

```diff
@@ -0,0 +1,61 @@
+import { CommunityCard, Footer, TopBar } from "components";
+import { NextPage } from "next";
+import Discord from "../icons/discord.svg";
+import GitHub from "../icons/github-icon.svg";
+import Twitter from "../icons/twitter.svg";
+import Patreon from "../icons/patreon.svg";
+import YouTube from "../icons/youtube.svg";
+interface CommunityProps {}
+const Community: NextPage<CommunityProps> = () => {
+  return (
+    <>
+      <TopBar />
+      <div className="home__image w-full h-40.4 bg-fixed items-center justify-center bg-no-repeat lg:bg-contain bg-cover bg-center md:bg-top lg:bg-top">
+        <div className="bg-opacity-70 flex flex-col bg-black-100 h-40.4 bg-fixed w-full">
+          <p className="text-white text-48 lg:text-72 self-center my-auto">
+            Community
+          </p>
+        </div>
+      </div>
+      <section className="bg-black-800 w-full px-32 grid grid-cols-1 sm:grid-cols-2 lg:grid-cols-4  gap-32 ">
```

Comment:
I believe we need a `CardsList` component instead of this section to use it here and in NewsList